// webpack-area-config-use/areaConfig.js
const APP_BUILD_USE_OSS = process.env.APP_BUILD_USE_OSS || false

// process.env.APP_BUILD_USE_OSS = APP_BUILD_USE_OSS.toString()

const testEnvRouterSuffix = APP_BUILD_USE_OSS ? 'test' : ''

const AREA_CONFIGS =
{

    // SERVER_PROXY:"http://192.168.1.235:8084",
    // SERVER_PROXY:"http://192.168.1.70:8080",
    SERVER_PROXY:"http://192.168.1.235:8084",
    IS_TEST_ENV: APP_BUILD_USE_OSS,


    zm_app: {
        APP_AREA: 'QY',
        APP_NAME: 'zm_app',
        APPID: '__UNI__000',
        APP_NAME_CN: '清远供水',
        MPWX_ID: 'wx619691b833c8ac4e',
        ROUTER_BASE: '/zm_app/h5' + testEnvRouterSuffix,
        PRODUCTION: {
            DEFAULT_ORIGIN: 'http://192.168.1.235:8084',
            BASE_URL: '',
            API_STR: '',
            FILE_OPEN_PREFIX: '',
        },
        DEV: {
            BASE_URL: '/api',
            API_STR: '/api',
            FILE_OPEN_PREFIX: '/api',
        },
        CONFIG: {

        }
    },
    zm_app_mp_dev: {
        APP_AREA: 'QY',
        APP_NAME: 'zm_app_mp_dev',
        APPID: '__UNI__000',
        APP_NAME_CN: '清远供水',
        MPWX_ID: 'wx619691b833c8ac4e',
        ROUTER_BASE: '/zm_app/h5' + testEnvRouterSuffix,
        PRODUCTION: {
            DEFAULT_ORIGIN: 'http://192.168.1.235:8084',
            BASE_URL: 'http://192.168.1.235:8084',
            API_STR: 'http://192.168.1.235:8084',
            FILE_OPEN_PREFIX: 'http://192.168.1.235:8084',
        },
        DEV: {
            BASE_URL: 'http://192.168.1.235:8084',
            API_STR: 'http://192.168.1.235:8084',
            FILE_OPEN_PREFIX: 'http://192.168.1.235:8084',
        },
        CONFIG: {
    
        }
    },
    zm_app_mp_local: {
        APP_AREA: 'QY',
        APP_NAME: 'zm_app_mp_local',
        APPID: '__UNI__000',
        APP_NAME_CN: '清远供水',
        MPWX_ID: 'wx619691b833c8ac4e',
        ROUTER_BASE: '/zm_app/h5' + testEnvRouterSuffix,
        PRODUCTION: {
            DEFAULT_ORIGIN: 'http://192.168.1.70:8080',
            BASE_URL: 'http://192.168.1.70:8080',
            API_STR: 'http://192.168.1.70:8080',
            FILE_OPEN_PREFIX: 'http://192.168.1.70:8080',
        },
        DEV: {
            BASE_URL: 'http://192.168.1.70:8080',
            API_STR: 'http://192.168.1.70:8080',
            FILE_OPEN_PREFIX: 'http://192.168.1.70:8080',
        },
        CONFIG: {
    
        }
    },
    zm_app_mp_pro: {
        APP_AREA: 'QY',
        APP_NAME: 'zm_app_mp_pro',
        APPID: '__UNI__000',
        APP_NAME_CN: '清远供水',
        MPWX_ID: 'wx619691b833c8ac4e',
        ROUTER_BASE: '/zm_app/h5' + testEnvRouterSuffix,
        PRODUCTION: {
            DEFAULT_ORIGIN: 'http://192.168.1.235:8084',
            BASE_URL: 'http://192.168.1.235:8084',
            API_STR: 'http://192.168.1.235:8084',
            FILE_OPEN_PREFIX: 'http://192.168.1.235:8084',
        },
        DEV: {
            BASE_URL: 'http://192.168.1.235:8084',
            API_STR: 'http://192.168.1.235:8084',
            FILE_OPEN_PREFIX: 'http://192.168.1.235:8084',
        },
        CONFIG: {

        }
    },

}

module.exports = AREA_CONFIGS
